features/box-decoration-break.yml.dist

```diff
@@ -0,0 +1,12 @@
+# Generated from: box-decoration-break.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
```

Comment:
this can't be right, the main property is supported everywhere. maybe add a `compute_from` for it and see if the resulting dist file changes? very strange that the other browsers don't show up at all though.

https://developer.mozilla.org/en-US/docs/Web/CSS/box-decoration-break#browser_compatibility

Reply:
The others aren't showing up because `box-decoration-break` is prefixed everywhere but Firefox.